modules/n1ql/pages/n1ql-language-reference/unnestposfun.adoc

```diff
@@ -0,0 +1,61 @@
+= UNNEST_POS Function
+:page-topic-type: reference
+:imagesdir: ../assets/images
+:description: Use the UNNEST_POS() function to return the position of each element in an unnested array.
+
+[abstract]
+{description}
+
+You can use the `UNNEST_POS()` function with the xref:n1ql-language-reference/unnest.adoc[UNNEST Clause] to return the position of each element in an unnested array. 
+
+[[fn-function-name,UNNEST_POS()]]
```

Comment:
needs a unique anchor. Doesn't really matter here but it will when it gets moved to `metafun.adoc`.

```suggestion
[[fn-unnest-pos,UNNEST_POS()]]
```